library/cwm/test/dialog_test.rb

```diff
@@ -32,5 +38,23 @@ def contents
 
       expect(subject.class.run).to eq(:launch)
     end
+
+    it "uses the back handler" do
```

Comment:
Does this actually test if the back handler is used? Is not the test itself the one calling it?